config/application.rb

```diff
@@ -33,5 +33,7 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+
+    config.VOLUNTEERS_AIRTABLE = "https://api.airtable.com/v0/appnZuBjB6ijeb3M2/all-volunteers"
```

Comment:
I think it'll be safer and neater to tuck this into an env variable, so this isn't revealed in our version control